src/gc/gcenv.ee.standalone.inl

```diff
@@ -11,166 +11,196 @@
 // will be fowarded to this interface instance.
 extern IGCToCLR* g_theGCToCLR;
 
+// A note about this:
+// In general, we don't want to pretend to be smarter than the compiler
+// and force it to inline things. However, inlining is here is required
+// for correctness as it stands today (though it will not always be required).
```

Comment:
Okay, that's what I figured - so then this will not be required at that time, when the GC is loaded dynamically?